scripts/update-contributions.js

```diff
@@ -17,37 +17,63 @@ Group.findAll({
   attributes: [
     'id',
     'name',
-    'settings'
+    'settings',
+    'data'
   ]
 })
   .tap(groups => {
     log.verbose('groups', `Found ${groups.length} group(s) to inspect`);
   })
   .each(group => {
     const org = _.get(group, 'settings.githubOrg');
-    if (!org) {
-      log.warn(group.name, `GitHub org not associated`);
+    const repoLink = _.get(group, 'settings.githubRepo');
+    if (!org && !repoLink) {
+      log.warn(group.name, `No GitHub org or repo associated`);
       return;
     }
-    return client.contributorsInOrg({orgs: [org]})
-      .get(org)
-      .then(repos => {
-      const data = {};
-      data.contributorData = _(repos)
-          .map('contributors')
-          .reduce((acc, contributions) => {
-            _.each(contributions, (count, user) => {
-              acc[user] = (acc[user] || 0) + count;
-            });
-            return acc;
-          }, {});
-        data.repoData = _.mapValues(repos, repo => _.omit(repo, 'contributors'));
-        return data;
-      })
-      .then(data => {
-        group.settings = _.assign(group.settings || {}, {
-          githubOrg: org
+    var fetchPromise;
+
+    if (org) {
+      fetchPromise = client.contributorsInOrg({orgs: [org]})
```

Comment:
with such a method name `contributorsInOrg`, I would expect the argument to be the string of the organization.